lib/Service/ProjectService.php

```diff
@@ -3613,6 +3616,112 @@ public function cronRepeatBills(?int $billId = null): array {
 		return $result;
 	}
 
+	private function copyBillPaymentMethodOver (string $projectid, array $bill, string $toprojectid): int {
+		$originPayments = $this->getCategoriesOrPaymentModes($projectid, false);
+		$destinationPayments = $this->getCategoriesOrPaymentModes($toprojectid, false);
+
+		if ($bill ['paymentmodeid'] != 0)
+		{
```

Comment:
```suggestion
		if ($bill ['paymentmodeid'] !== 0) {
```
* Negation of `===` is `!==`.
* Let's keep the opening curly bracket on the same line.

Reply:
Whops, those brackets slipped, should be fixed now everywhere in the code